prac_05/state_names.py

```diff
@@ -5,14 +5,17 @@
 """
 
 # TODO: Reformat this file so the dictionary code follows PEP 8 convention
-CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT" : "Northern Territory", "WA" : "Western Australia",
-            "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
 print(CODE_TO_NAME)
+for state_short_name, state_name in CODE_TO_NAME.items():
```

Comment:
This code looks good.
I recommend changing the name "state_short_name" to something else (like "state_abbreviation" or "state_code"). I feel that it would be more readable that way, though I think what you have is perfectly functional.